### PR TITLE
Recommendation

### PR DESCRIPTION
**Pull request by songhappy**
Publish recommendation notebook
Tested under BigDL 0.3 with spark 2.0 and 2.1

### CODE REVIEW

File: notebook/recommendation/recommender-explicit-feedback.ipynb [hunk 1]
```diff
@@ -0,0 +1,694 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "* Understand the data\n",
+    "\n",
+    "Each record is in format of (userid, movieid, rating_score). \n",
+    "UserIDs range between 1 and 6040 \n",
```
Comment (jason-dai) [inline notebook/recommendation/recommender-explicit-feedback.ipynb]: The newline mark "\n" does nor work when rendering on github

File: notebook/recommendation/recommender-explicit-feedback.ipynb [hunk 2]
```diff
@@ -0,0 +1,694 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prediction\n",
+    "\n",
+    "* BigDL models make inferences based on the given data using model.predict(val_rdd) API. A result of RDD is returned. predict_class returns the predicted label. \n",
+    "* Example of usecase: given some users and movies, we can recommend  a list of movies(here topK =3) for the users. We use user ids and movie ids from validation data set as example to show the recommendation.  uidTopKMovies is defined as a list of key value pairs, where key is user id, value is a list of top movie id and predicted label."
```
Comment (jason-dai) [inline notebook/recommendation/recommender-explicit-feedback.ipynb]: Example of use case

Comment (jason-dai) [reply notebook/recommendation/recommender-explicit-feedback.ipynb]: value is a list of topK movie id and predicted label (rating)

Comment (jason-dai) [reply notebook/recommendation/recommender-explicit-feedback.ipynb]: Please add sample output of print(uidTopKMovies[0:10])

**Comment by jason-dai** [comment]
The notebook folder is renamed to apps in https://github.com/intel-analytics/analytics-zoo/pull/47

**Comment by jason-dai** [comment]
both notebook and apps folders exist in your PR

**Comment by songhappy** [comment]
fixed, and ready to merge

File: README.md [hunk 3]
```diff
@@ -1,2 +1,2 @@
-# Analytics Zoo
+#  Analytics Zoo
```
Comment (jason-dai) [inline README.md]: Why the change?

**Comment by jason-dai** [comment]
Why are there changes to files other hand the recommendation notebook?

**Comment by songhappy** [comment]
I simply wanted to rename the folder, and it caused some unintentional change, fixed it by adding everything from the new master

**Comment by jason-dai** [comment]
LGTM
